hebiros_description/urdf/omnibase.xacro

```diff
@@ -0,0 +1,170 @@
+    <origin xyz="-0.125 0.0 0.030" rpy="0 1.5708 3.14159"/>
+    <axis xyz="0 0 1"/>
+  </joint>
+
+
+
+  <link name="HEBI/Module1">
+    <visual>
+      <origin xyz="0 0 0.030" rpy="0 0 0"/>
+      <geometry>
+        <!--box size="0.50 0.50 0.10"/-->
+        <mesh filename="package://hebiros_description/meshes/x5.stl" scale="0.001 0.001 0.001"/>
+      </geometry>
+      <xacro:red_material/>
+    </visual>
+
+  </link>
+
+
+  <link name="HEBI/Module2">
+    <visual>
+      <origin xyz="0 0 0.030" rpy="0 0 0"/>
+      <geometry>
+        <!--box size="0.50 0.50 0.10"/-->
+        <mesh filename="package://hebiros_description/meshes/x5.stl" scale="0.001 0.001 0.001"/>
+      </geometry>
+      <xacro:red_material/>
+    </visual>
+  </link>  
+
+  <link name="HEBI/Module3">
+    <visual>
+      <origin xyz="0 0 .030 " rpy="0 0 0"/>
+      <geometry>
+        <!--box size="0.50 0.50 0.10"/-->
+        <mesh filename="package://hebiros_description/meshes/x5.stl" scale="0.001 0.001 0.001"/>
+      </geometry>
+      <xacro:red_material/>
+    </visual>
+  </link>
+
+  <joint name="omniwheel1_joint" type="fixed">
+    <parent link="HEBI/Module1"/>
+    <child link="omniwheel1"/>
+    <origin xyz="0 0 .060" rpy="0 0 0"/>
+    <axis xyz="0 0 1"/>
+  </joint>
+
+
+  <joint name="omniwheel2_joint" type="fixed">
+    <parent link="HEBI/Module2"/>
+    <child link="omniwheel2"/>
+    <origin xyz="0 0 0.060" rpy="0 0 0"/>
+    <axis xyz="0 0 1"/>
+  </joint>
+
+
+  <joint name="omniwheel3_joint" type="fixed">
+    <parent link="HEBI/Module3"/>
+    <child link="omniwheel3"/>
+    <origin xyz="0.0 0.0 0.060" rpy="0 0 0"/>
+    <axis xyz="0 0 1"/>
+  </joint>
+
+
+  <link name="omniwheel1">
+    <visual>
+      <origin xyz="0 0 0.030" rpy="0 0 0"/>
+      <geometry>
+        <mesh filename="package://hebiros_description/meshes/omniwheels.stl" scale="0.001 0.001 0.001"/>
+      </geometry>
+      <material name="light_black"/>
+    </visual>
+
+  </link>
+
+
+  <link name="omniwheel2">
+    <visual>
+      <origin xyz="0 0 0.030" rpy="0 0 0"/>
+      <geometry>
+        <mesh filename="package://hebiros_description/meshes/omniwheels.stl" scale="0.001 0.001 0.001"/>
+      </geometry>
+      <material name="light_black"/>
+    </visual>
+  </link>  
+
+  <link name="omniwheel3">
+    <visual>
+      <origin xyz="0 0 .030 " rpy="0 0 0"/>
+      <geometry>
+        <mesh filename="package://hebiros_description/meshes/omniwheels.stl" scale="0.001 0.001 0.001"/>
+      </geometry>
+      <material name="light_black"/>
+    </visual>
+  </link>
+
+
+  <!-- LIDAR Information-->
+
+  <joint name="scan_joint" type="fixed">
+    <parent link="base_link"/>
+    <child link="base_scan"/>
+    <origin xyz="0 0 0.180" rpy="0 0 1.5708"/>
+  </joint>
+
+  <link name="base_scan">
+    <visual>
+      <origin xyz="0 0 0" rpy="0 0 0"/>
+      <geometry>
+        <mesh filename="package://hebiros_description/meshes/lds.stl" scale="0.001 0.001 0.001"/>
```

Comment:
Why is the STL file just called `lds`?  Does that stand for something in particular (perhaps laser depth scanner)?

I'd rename the file to make it clear what it is _and_ add a model name/description.

Also, just wanted to check b/c I don't know where you got the file -- is there any attribution we need to give or copyright that would prevent us from using this in our repository?